ports/voidstranger/voidstranger/game_patching.txt

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# get data.win checksum
+if [ -f "gamedata/data.win" ]; then
+	game_chksm=$(md5sum gamedata/"data.win" | awk '{print $1}')
+else
+	echo "WARNING: Game data not found; check the instructions."
+	return 1
```

Comment:
Bash supports returns this way since the script is sourced. In Ash, this should be `exit 1` instead. In our discussion yesterday I had not considered ash.

https://www.geeksforgeeks.org/difference-between-ash-and-bash/

Reply:
changed

Reply:
waiting to test on trimui before commit